### ISSUE TITLE
Update README with Docker + Binder instructions

### ISSUE THREAD

**Issue by edwenger** [edwenger/emod-demo#3]

**Comment by edwenger** [comment]
Helpful template here: https://raw.githubusercontent.com/fastscape-lem/fastscape-demo/master/README.md

**Comment by edwenger** [comment]
after #9, local docker no longer relevant part of use pattern
